### PR TITLE
Add export off in DGtal::Shortcut

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
To simplify export and view inside Tools of DGtalTools. More basic but used in current DGtatTools PR  https://github.com/DGtal-team/DGtalTools/pull/370

# Checklist

- [x] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

File: src/DGtal/helpers/Shortcuts.h [hunk 1]
```diff
@@ -1651,6 +1653,67 @@ namespace DGtal
...
+      /// Outputs a digital surface as an OFF file (with its
+      /// topology).  Optionnaly you can specify the face colors (see
+      /// saveOBJ for a more advanced export).
+      ///
+      /// @tparam TDigitalSurfaceContainer any model of concepts::CDigitalSurfaceContainer
+      /// @tparam TCellEmbedder any type for maping Cell -> RealPoint.
+      ///
+      /// @param[in] digsurf the digital surface to output as an OBJ file
+      /// @param[in] embedder any map Cell->RealPoint
+      /// @param[in] face_colors either empty or a vector of size `trisurf.nbFaces` specifying the diffuse color for each face.
```
Comment (JacquesOlivierLachaud) [inline src/DGtal/helpers/Shortcuts.h:1665]: Seeing the type face_colors should be face_color and is only one color. Or you have to update the code below to handle a vector of Colors.

File: src/DGtal/helpers/Shortcuts.h [hunk 2]
```diff
@@ -1651,6 +1653,67 @@ namespace DGtal
...
+              auto primal_vtcs = getPointelRange( K, surfel );
+              output_off << primal_vtcs.size();
+                {
+                  for ( auto&& primal_vtx : primal_vtcs )
+                    output_off << " " << (c2i[ primal_vtx ]);
+                }
+                if(face_color != DGtal::Color::None)
+                {
+                  output_off << " ";
+                  output_off  << ((double) face_color.red())/255.0 << " "
```
Comment (JacquesOlivierLachaud) [inline src/DGtal/helpers/Shortcuts.h:1708]: We should add methods r() g() b() a() in Color to return a double in [0,1]

File: src/DGtal/helpers/Shortcuts.h [hunk 3]
```diff
@@ -2292,6 +2378,33 @@ namespace DGtal
           return ok;
         }
 
+       /// Outputs a triangulated surface as an OFF file.
+      /// 
+      ///
+      /// @tparam TPoint any model of point
+      /// @param[in] polysurf the polygonal surface to output as an OBJ file
+      /// @param[in] off_file the output filename.
+      /// @param[in] face_color the comming face color.
```
Comment (JacquesOlivierLachaud) [inline src/DGtal/helpers/Shortcuts.h:2387]: the color of every face ?

File: src/DGtal/helpers/Shortcuts.h (hunk 1 above)
Comment (kerautret) [reply src/DGtal/helpers/Shortcuts.h:1665]: Thanks I forget ;)

File: src/DGtal/helpers/Shortcuts.h (hunk 2 above)
Comment (kerautret) [reply src/DGtal/helpers/Shortcuts.h:1708]: yes done

File: src/DGtal/helpers/Shortcuts.h (hunk 3 above)
Comment (kerautret) [reply src/DGtal/helpers/Shortcuts.h:2387]: yes thanks

**Comment by kerautret** [comment]
> Apart from the few typos, looks ok to me. I assume you have checked that the OFF works.

Thanks @JacquesOlivierLachaud, corrections done. Yes this PR is used in the PR tools that I used it to produce 3d illustrations. ;)

**Comment by JacquesOlivierLachaud** [comment]
All good then ! Merging.

**Pull request merged**